### DISCUSSION TITLE
最新版本5.15局域网传输会导致断线重连

### DISCUSSION THREAD

**Discussion by aa889788** [Boos4721/openwrt#123] [Bugs/问题反馈]
使用的是作者发布的最新版本，在局域网中有一个nas，在其中跑了一个android虚拟机，平常使用scrcpy连接上去玩手游，升级5.15内核之后，连接android虚拟机几分钟后即会导致断线重连，在日志中也找不到相关信息
[syslog.log](https://github.com/Boos4721/openwrt/files/8545796/syslog.log)
[kernel.log](https://github.com/Boos4721/openwrt/files/8545799/kernel.log)
而且目前尝试降级到5.10，不论是使用`sysupgrade -F -n`，还是在web选择不保留配置升级，在重启之后仍然是原系统，不清楚如何解决

**Comment by Boos4721** [comment]
https://github.com/Boos4721/openwrt/discussions/9

**Reply by aa889788** [reply]
> #9

已经成功回退，回退后没有再出现断线重连问题
